### DISCUSSION TITLE
post/learn-about-use-id/

### DISCUSSION THREAD

**Discussion by giscus** [jgjgill/blog#8] [Comments]
# post/learn-about-use-id/

useId와 관련된 개념들을 이해합니다.

https://jgjgill.com/post/learn-about-use-id/

**Comment by JunilHwang** [comment]
> 한 페이지에서 여러 개의 독립적인 React 애플리케이션을 렌더링하는 상황이 발생할 수도 있다. (아직 한 번도 경험해보지 못했지만... 😇)

"마이크로 프론트엔드" 라고 이야기한답니다! ㅋㅋ

**Reply by jgjgill** [reply]
오오.. 이렇게 또 하나 배우네요.. 감사합니다..! 🙇‍♂️
